testing/integration/README.md

```diff
@@ -0,0 +1,52 @@
+# Integration tests
+
+These Python-based tests are meant to test the fully-deployed system from a
+black-box point of view, and the goal is to test every public-facing endpoint.
+
```

Comment:
I'm *assuming* we're not going to actually automate the integration tests, right? The initial plan is that we (you, me, etc) just keep it in our back pocket to run manually when we feel like it. 

I think you should mention something about that here. It certainly was my first question in my head: "I know what they do, but why do they exist and who/what runs them?"

Which brings to mind, would there be a reason NOT to automatically and/or periodically running the suite?
If we put it at the bottom, for example, of `stage-build.yml` it won't block the stage build even if something was to fail. The worst that could happen is that we get an email from GitHub Actions saying the workflow failed and we might get a bit confused like "Well, did it manage to deploy at least?"

Perhaps it's best to simply "replicate" what we did with `prod-build.yml` and `stage-build.yml` and have the same cron config but have it just run these tests.

Reply:
@peterbe Great points and questions. I initially had wanted to just get these into the repo so we all could use them manually for now, and left the question of automation open. However, I think I like best the idea of running them as separate stage and prod integration-tests-only cron workflows, since we might want to run them more frequently, like every hour, like a system check heartbeat. They take less than 2 minutes to run.
I'll leave that for a separate PR.